dubbo-common/src/main/java/org/apache/dubbo/config/ApplicationConfig.java

```diff
@@ -203,6 +204,12 @@ public class ApplicationConfig extends AbstractConfig {
 
     private Boolean enableEmptyProtection;
 
+    /**
+     * enable mesh mode
+     * @since 3.1.0
+     */
+    private Boolean meshEnable;
```

Comment:
`meshEnable` should be the behavior of the consumer and should be placed in the configuration of the consumer level